### PR TITLE
misc cleanups

### LINKED ISSUE #16364: `&` shows as `&amp;` in docs
### Example
in https://nim-lang.github.io/Nim/lib.html search for `&`

### Current Output

image

### Expected Output
show `&`, as it did in stable docs 1.4.x

image

### Additional Information
devel 1.5.1 e843492b1334ded1d8fcd0e0eb6dc94fcf970aba
there could be more such symbols

### PR DESCRIPTION
**Pull request by timotheecour**
fixes https://github.com/nim-lang/Nim/issues/16364#issuecomment-747617572 + other cleanups

### CODE REVIEW

File: compiler/modulegraphs.nim [hunk 1]
```diff
@@ -28,7 +28,7 @@
 import ast, intsets, tables, options, lineinfos, hashes, idents,
   incremental, btrees, md5
 
-import ic / packed_ast
+# import ic / packed_ast
```
Comment (ringabout) [inline compiler/modulegraphs.nim:31]: Why comment this?

Comment (timotheecour) [reply compiler/modulegraphs.nim:31]: to silence `UnusedImport` warning; I can also just remove this line if you prefer

**Review by ringabout — APPROVED** [review]

File: compiler/modulegraphs.nim (hunk 1 above)
Comment (ringabout) [reply compiler/modulegraphs.nim:31]: Maybe don't handle UnusedImport warning it in this PR, `IC` changed rapidly I guess.

Comment (timotheecour) [reply compiler/modulegraphs.nim:31]: I mean it's easy enough to uncomment this line if it ever becomes needed :) 
also helps reduce risk of cyclic imports

**Pull request merged**